dds_web/api/files.py

```diff
@@ -527,11 +527,12 @@ def delete_folder(self, project, folder):
                 .filter(
                     sqlalchemy.or_(
                         models.File.subpath == sqlalchemy.func.binary(folder),
-                        models.File.subpath.regexp_match(rf"^{folder}(/[^/]+)*$"),
+                        models.File.subpath.regexp_match(rf"^{folder}(\/[^\/]+)*$"),
```

Comment:
Doesn't this imply that there needs to be `folder/` and then something else? The current solution allows for both `folder` and `folder/` to be put in and match the same thing. 

Yes regex has been discussed a lot. This PR is only to fix the `rm` functionality since `rf"^{folder}(/[^/]+)*$"` doesn't match _anything_, but after escaping the `/` it does.

Can you elaborate on why it would be cleaner?

Reply:
Yes, indeed. If you want `folder` and `folder/ `to be matched as well and not only the folder's contents, then it must read `rf"^{folder}(?:\/?[\w \.]?\/?)+$"`